src/__experimental__/components/checkbox/checkbox.component.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import tagComponent from '../../../utils/helpers/tags';
+import { validProps } from '../../../utils/ether';
+import FormField from '../form-field';
+import CheckboxStyle from './checkbox.style';
+
+const Checkbox = (props) => {
+  const formFieldProps = () => {
+    const { ...validFormFieldProps } = validProps(
+      { propTypes: Checkbox.propTypes, safeProps: ['fieldHelpInline'], props }
+    );
+
+    validFormFieldProps.reverse = !props.reverse;
+
+    return validFormFieldProps;
+  };
+
+  const inputProps = () => {
```

Comment:
Why does this need to be a function? If you assign anonymous functions inside the body of the component (which is itself a function), it will be re-defined every time the component is rendered. 

Better to define functions outside the scope of the parent function, if they're needed at all.

Reply:
Agreed, this didn't really need to be a function - update 👍